cfgov/v1/jinja2/v1/includes/molecules/translation-links.html

```diff
@@ -28,20 +26,15 @@
 
     ======================================================================== #}
 
-{% macro render(value={}) %}
-
-    {%- if page is defined and not value.language or not value.links %}
-        {% set temp = value.update({
-            'language': ( language or page.language )| default( "en" ),
-            'links': page.get_translation_links( request ),
-        }) %}
-    {%- endif %}
+{% macro render(links=None, modifier_classes="") %}
```

Comment:
Per [Jinja documentation](https://jinja.palletsprojects.com/en/3.0.x/templates/#literals), this can be `None` or `none` but they recommend using `none`.